config/pure-python-without-pypy/gitignore

```diff
@@ -0,0 +1,14 @@
+*.egg-info
+*.pyc
+*.pyo
```

Comment:
I suggest also adding `__pycache__/`, although the above rules would probably pick everything under it anyway.

Reply:
I think `__pycache__` is Python 3 only.